source/includes/steps-tutorial-unity-sync.yaml

```
@@ -0,0 +1,207 @@
+---
+title: Enable Realm Sync
+ref: enable-realm-sync
+content: |
+  Before you can add {+sync-short+} functionality within your game, you must
+  enable {+sync+} in your {+backend+} app. 
+
+  To enable {+sync+}, follow the steps in the :doc:`Get Started with Sync
+  </sync/get-started>` guide. When enabling {+sync-short+}, you will need to configure your app to:
+
+  - use ``Development Mode``. 
+  - add a required :doc:`Partition Key </sync/partitions>` with the name ``"_partition"`` and with the type ``string``.
```

Given our ongoing IA updates, it's probably better to use a `:ref:` here vs. a `:doc:` link.

```suggestion
  - add a required :ref:`Partition Key <sync-partitions>` with the name ``"_partition"`` and with the type ``string``.
```